flaskbb/user/__init__.py

```diff
@@ -1,3 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
Turns out we never had this header and I didn't add it when I went through and added all the loggers.